Lib/test/test_unicode.py

```diff
@@ -2880,10 +2989,11 @@ def check_format(expected, format, *args):
         # check for crashes
```

Comment:
this comment is misleading. Python doesn't crash but reject invalid format string with a SyntaxError. Would you mind to rephrase the comment? Like: ``# test invalid format strings``?

Reply:
Done. Restored the part of the older comment.